packages/typescript-estree/src/create-program/createIsolatedProgram.ts

```diff
@@ -66,6 +66,7 @@ function createIsolatedProgram(parseSettings: ParseSettings): ASTAndProgram {
       noResolve: true,
       target: ts.ScriptTarget.Latest,
       jsx: parseSettings.jsx ? ts.JsxEmit.Preserve : undefined,
+      experimentalDecorators: true,
```

Comment:
For passing tests and avoid breaking change.
But ideally, I think we should have new option `experimentalDecorators`: https://github.com/typescript-eslint/typescript-eslint/issues/6571

Reply:
Do we need to set this at all? As per the issue - if we leave this as `false` ts will just emit semantic errors (which we ignore)

Reply:
Fixed :+1: [1210fec](https://github.com/typescript-eslint/typescript-eslint/pull/6570/commits/1210fec96805b89c436b87eb5fe1569034d9ce84)